docs/PdfExportOverview.md

```diff
@@ -1,111 +1,43 @@
-## PDF Export for SurveyJS
+# SurveyJS PDF Export Overview
 
-SurveyJS PDF exporter library is an easy way to render [SurveyJS Library](https://surveyjs.io/Overview/Library/) surveys to PDF which can be emailed or printed.
+SurveyJS PDF Export is a client-side extension over the [SurveyJS Library](/Documentation/Library) that enables users to save surveys as PDF documents.
```

Comment:
SurveyJS PDF Export is a client-side extension over the [SurveyJS Library](/Documentation/Library), which enables users to save surveys as PDF documents.